src/ArmTemplates/Extractor/EntityExtractors/ProductApisExtractor.cs

```diff
@@ -82,12 +90,12 @@ async Task<List<TemplateResource>> GenerateSingleApiTemplateAsync(string singleA
             }
         }
 
-        async Task<List<TemplateResource>> GenerateMultipleApisTemplateAsync(List<string> multipleApiNames, ExtractorParameters extractorParameters)
+        async Task<List<ProductApiTemplateResource>> GenerateMultipleApisTemplateAsync(List<string> multipleApiNames, ExtractorParameters extractorParameters)
         {
             this.logger.LogInformation("Processing {0} api-names...", multipleApiNames.Count);
 
             string[] dependsOn = new string[] { };
```

Comment:
```suggestion
            string[] dependsOn = Array.Empty<string>();
```

Can you do this everywhere where you create an empty array?

Reply:
tried to change all extractor appearences